backend/src/cusco/sources/prr.py

```
@@ -0,0 +1,365 @@
+                    if not url:
+                        continue
+                    if url_l.endswith(".xlsx") and (
+                        hint_l in title or hint_l in url_l
+                    ):
+                        return url
+                # Fall back to the first xlsx resource if no hint match
+                for resource in dataset.get("resources", []):
+                    url = resource.get("url") or ""
+                    if url.lower().endswith(".xlsx"):
+                        return url
+        except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)